README.md

```diff
@@ -1,28 +1,14 @@
-# Portfolio-setup-and-mobile-first
+# Alejandra Keber - Portfolio
 
-Description of the project:
-- Set up a new repository and prepare it for development using best practices (e.g. linters).
-- Build the first 2 sections of the mobile website using the template you chose in the previous step.
+Portfolio based in Figma Template 1
 
 **Built With**
 Javascript, HTML and CSS
 
 **Getting started:**
-- Create a new repository in Github
-- In your local machine run the comand: git clone Repository's URL
-- Create a feature branch usin the comand: git branch FeatureBranch
-- Work in the feature branch usin the command: git checkout FeatureBranch
-- Open the repository in your editor (Visual studio code or Atom)
-- Add changes and commits and push them to the repository usin: git push -u origin FeatureBranch
-
-**General Requirements:**
-- Make sure that there are no linter errors
-- Make sure that you used correct GitHub Flow
-- Make sure that you documented your work in a professional way
-
-**Project Requirements:**
-- For this project, you should use the HTML&CSS linter
-- The 2 sections you need to build for this project are:  the tool bar and the headline section
+- Copy the URL code
+- Git clone to your local machine
+- Open the folder with you code editor
```

Comment:
- [ ] Please add the following commands in your Getting Started Section. :+1: because it is your readme file it should be descriptive :+1:
```
git clone git@github.com:AlejandraKeber/Project-setup-and-mobile-main.git
cd Project-setup-and-mobile-main
```